### PR TITLE
Update Ads debug fields for consistency

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue:

- #8245 

## Relevant technical choices

See below

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: includes/Modules/Ads.php [hunk 1]
```diff
@@ -173,9 +174,10 @@ public function get_debug_fields() {
 		$settings = $this->get_settings()->get();
 
 		return array(
-			'ads_conversion_id' => array(
-				'label' => __( 'Conversion Tracking ID', 'google-site-kit' ),
+			'ads_conversion_tracking_id' => array(
+				'label' => __( 'Ads Conversion Tracking ID', 'google-site-kit' ),
```
Comment (aaemnnosttv) [inline includes/Modules/Ads.php:178]: These should be prefixed with the module name and generally serve the same purpose in different places (label is for HTML output, key is for clipboard/debug).

File: includes/Modules/Ads.php [hunk 2]
```diff
@@ -173,9 +174,10 @@ public function get_debug_fields() {
...
 				'value' => $settings['adsConversionID'],
+				'debug' => Debug_Data::redact_debug_value( $settings['adsConversionID'] ),
```
Comment (aaemnnosttv) [inline includes/Modules/Ads.php:180]: This is how IDs are formatted in debug data (clipboard). See GA for comparison https://github.com/google/site-kit-wp/blob/029750d5e5394cefd9dac2a971fb51da76d90d28/includes/Modules/Analytics_4.php#L401

File: includes/Modules/Analytics_4.php [hunk 3]
```diff
@@ -396,7 +396,7 @@ public function get_debug_fields() {
 				'debug' => Debug_Data::redact_debug_value( $settings['accountID'] ),
 			),
 			'analytics_4_ads_conversion_id'           => array(
-				'label' => __( 'Analytics ads conversion ID', 'google-site-kit' ),
+				'label' => __( 'Analytics Ads conversion ID', 'google-site-kit' ),
```
Comment (aaemnnosttv) [inline includes/Modules/Analytics_4.php:399]: Our use of case is inconsistent in the labels, but product names should be capitalized.

**Review by kuasha420 — APPROVED** [review]
Thanks @aaemnnosttv ! LGTM :tada:

**Pull request merged**